### DISCUSSION TITLE
Plotting multiple sets of grain boundaries on one map

### DISCUSSION THREAD

**Discussion by alexcaustin** [mtex-toolbox/mtex#2175] [Ask Anything]
Hello,
I am trying to plot grain boundary traces calculated in two different way on top of each other on an ipf map. I have used OIM to calculate these segments but want to use MTex to make the plots. Is there a way of plotting reconstructed boundary segemnts exported from OIM (example file attached) on top of an ipf map? 

Thanks!
[recon_boundaries.txt](https://github.com/user-attachments/files/16347836/recon_boundaries.txt)

**Comment by kilir** [comment]
Hi,

Mtex does not currently have an interface for such kind of data. Probably easiest would be if you can export your segmented EBSD data from your software with a grain Id such that you can reconstruct grains in mtex based on that id.

Cheers,

Rüdiger

**Comment by alexcaustin** [comment]
Hi Rüdiger,

Thanks for getting back to me, that's good to know. I will give this a try.

I have another, unrelated question. I'm trying to plot a full GBCD on an equal angle plot. I know the default in MTex is equal area but I don't seem to be able to change this.
I have tried 'plotGBPD(gbpd,cs,'projection','stereo')' and the 'eangle' command but haven't had any luck.

Thanks for your help,
Alex

________________________________

**Reply by ralfHielscher** [reply]
Hi Alex,

could you please post your question in a separate post?

Thank you very much,
Ralf.

**Reply by alexcaustin** [reply]
Hi Ralf, 

Okay, will do!

Thanks,
Alex
